org.js.nuclear.Nuclear.json

```diff
@@ -31,13 +31,13 @@
                 },
                 {
                     "type":"file",
-                    "url":"https://raw.githubusercontent.com/advaithm/nuclear/master/flatpak/org.js.nuclear.Nuclear.metainfo.xml",
+                    "url":"https://raw.githubusercontent.com/advaithm/flathub/nuclear/org.js.nuclear.Nuclear.metainfo.xml",
```

Comment:
use the pah:
```json
"path": "org.js.nuclear.Nuclear.metainfo.xml"
```

Reply:
This is not what I meant. Remove `url`, path is enough. Also since the file is now local, you don't need the `sha256`

Reply:
I said to remove `url` too.